### DISCUSSION TITLE
[✨] SST Integration - Adapter configuration for Qwik City

### DISCUSSION THREAD

**Discussion by thejackshelton** [QwikDev/qwik-evolution#155] [Proposals For Qwik]
### Is your feature request related to a problem?

For the past several weeks We've been conversing with the devs over at SST to try and get an integration with Qwik City.

Along the way some fellow Qwik community members have joined in. Including @KenAKAFrosty

SST is a thin layer between AWS that allows you to add any AWS service, db, auth, api, etc.

Frank is an awesome developer over at SST that recently finished a Svelte integration and started working on one with Qwik City for us.

You can find our recent conversation here. 
https://discord.com/channels/842438759945601056/991805589623668758/1106591648990887996

I can also toss a discord invite to the channel where we've been conversing in SST.

### Describe the solution you'd like

The ability to have an sst-adapter in a separate repo for Qwik City. Along with SST added as an option for the deployments section of the Qwik ecosystem.

### Describe alternatives you've considered

Frank considered having the adapter in the same repo, but found that it would slow down the process of finding/fixing bugs.

> Hit into a blocker... it seems adapters need to live within qwik's repo. That's going to make the adapter a bit tricky to maintain and push out quick fixes ⁠Builder.io⁠ 
Do u have an example of any "community-managed adapters" that lives in independent repos? 
^ to add some context, we felt this w/ Astro. Originally, our Astro adapter lived in Astro's repo, and bug fixes took weeks to get approved.

@adamdbradley have any insights or guidance that I can relay to Frank? An integration here would be awesome for people deploying to AWS with Qwik 

### Additional context

After finding out how powerful SST was (and that it's open source), I saw huge potential for an integration with Qwik.

Especially since Qwik apps are instantly interactive and performant at scale, I think both projects align well together for company and enterprise level software development.

I was excited to see that most people in the community agree. Including @steve8708!

If there's anything I can do to help get this one step further let me know!

**Comment by NicoPowers** [comment]
Any updates on this? This would be amazing, been using SST and since its built on top of AWS-CDK, developers can literally build ANY kind of infrastructure with SST + AWS-CDK. No more switching between different infrastructure platforms that just leverage AWS under the hood anyways, and don't even have all the features one may need.

**Comment by thejackshelton** [comment]
Hey @NicoPowers! No updates unfortunately. I believe there needs to be some knowledge from the SST side on how to setup the adapter. I remember Frank from the SST team looked at it for a little bit but was super busy. If you're interested I would reach out to him about contributing to a Qwik integration.

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.
